lib/tasks/transcribe.js

```diff
@@ -175,8 +175,8 @@ class TaskTranscribe extends SttTask {
   }
 
   async _setSpeechHandlers(cs, ep, channel) {
-    if (this._speechHandlersSet) return;
-    this._speechHandlersSet = true;
+    if (ep.speechHandlersSet) return;
```

Comment:
please rename to `ep._speechHandlersSet`

Reply:
fixed